docs/tutorials/groupby_doc.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>Given the amount of ways that this can fail or be weird to a given use case, I would love to see a small note at the end here that encourages users to make issues if they're trying to use this in nested-pandas and running into issues!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-pandas/pull/396/discussion/#comment-2487495168'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>